test/test_structured_1d.jl

```diff
@@ -20,6 +20,12 @@ isdir(outdir) && rm(outdir, recursive=true)
       linf = [3.217887726258972e-5])
   end
 
+  @trixi_testset "elixir_advection_shockcapturing.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_shockcapturing.jl"),
+      l2   = [0.0630105298116176],
+      linf = [0.5884019851242106])
+  end
```

Comment:
This test fails in CI, see https://github.com/trixi-framework/Trixi.jl/actions/runs/4433677116/jobs/7778949296#step:7:1886

Reply:
On my computer, I do get the numbers that I had earlier put. I have now replaced them with the ones that CI expected.